src/core/standard_fonts.js

```diff
@@ -305,6 +305,93 @@ const getSymbolsFonts = getLookupTableFactory(function (t) {
   t.ZapfDingbats = true;
 });
 
+const getStdFontMetrics = getLookupTableFactory(function (t) {
```

Comment:
This honestly doesn't look exactly like data that belong in this file, and you probably want to use https://github.com/mozilla/pdf.js/blob/master/src/core/metrics.js instead.

Also, please let's improve the sorting of this data such that the various font-families have their data grouped together instead. I.e. something like:
```javascript
t["Times-Roman"] = { ... };
t["Times-Bold"] = { ... };
t["Times-BoldItalic"] = { ... };
t["Times-Italic"] = { ... };
t.Helvetica = { ... };
t["Helvetica-Bold"] = { ... };
...
```

Reply:
I just copied/pasted the code here:
https://github.com/mozilla/pdf.js/blob/6b9cc24d49583c487d9267c98579fd4cd9db1704/src/core/standard_fonts.js#L23-L38